src/MessagingBubble/MessagingBubble.less

```diff
@@ -22,6 +22,42 @@
   .items--start();
 }
 
+.MessagingBubble--Message--Container--Own.MessagingBubble--FamilyPortal {
+  .MessagingBubble--Message--Own {
+    background-color: @family_portal_own_message;
+    border-radius: 1rem;
+    color: @family_portal_slate;
+    font-weight: 400;
+    .padding--x--m();
+    .padding--y--xs();
+  }
+
+  .MessagingBubble--Message--Timestamp--Own {
+    margin-right: @size_m;
+  }
+
+  .MessagingBubble--Message--Reply--Own {
+    // TODO: is this the best way to override a background-color?
+    background-color: unset;
```

Comment:
this makes sense to me since it's inheriting the background color from `.MessagingBubble--Message--Own`!